src/demo/coupons.js

```diff
@@ -119,18 +119,17 @@ const getAllCodes = (couponHasError = false) => [...Array(codesCount)].map((_, i
 const allCodes = getAllCodes();
 
 const unassignedCodes = allCodes.filter(code => (
-  code.redemptions.available > minRedemptionsCount && code.redemptions.used < maxRedemptionsCount
+  code.assigned_to === undefined && code.redemptions.used < code.redemptions.available
 ));
 const unredeemedCodes = allCodes.filter(code => (
-  (code.redemptions.available === minRedemptionsCount &&
-    code.redemptions.used > minRedemptionsCount && code.redemptions.used < maxRedemptionsCount)
+  code.assigned_to !== undefined && code.redemptions.used < code.redemptions.available
 ));
 const partiallyRedeemedCodes = allCodes.filter(code => (
-  code.redemptions.available > minRedemptionsCount && code.redemptions.used > minRedemptionsCount
+  (code.redemptions.used > minRedemptionsCount &&
```

Comment:
**Note to reviewers:** Ideally this should be `code.redemptions.used => minRedemptionsCount` (at-least 1 redemption) but for the sake of demo data, `partiallyRedeemedCodes` need to be different than `unRedeemedCodes`, we had to do this (since there is no distinction in frontend demo data for differentiating unredeemed/redeemed assignments and redemptions used).